src/base/memory_monitor_sr.cpp

```diff
@@ -557,16 +724,44 @@ void mmon_resize_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, int64_t old
     }
 }
 
-int mmon_aggregate_module_info (MMON_MODULE_INFO *info, int module_index)
+int mmon_aggregate_module_info (MMON_MODULE_INFO &info, int module_index)
+{
+  int error = NO_ERROR;
+
+  assert (mmon_Gl != nullptr);
+  assert (module_index < (int)MMON_MODULE_LAST && module_index > 0);
+
+  error = mmon_Gl->aggregate_module_info (info, module_index);
+
+  return error;
+}
+
+int mmon_aggregate_module_brief_info (MMON_MODULE_INFO *&info, int option)
 {
   int error = NO_ERROR;
 
+  assert (mmon_Gl != nullptr);
+
+  if (option == MMON_MODULE_BRIEF_OPTION || option == MMON_MODULE_DEFAULT_OPTION)
```

Comment:
Please elaborate on the need of `option`. `MMON_MODULE_BRIEF_OPTION` is not used under `aggregate_module_brief_info`, I can't even guess it.

Reply:
In memmon utility, there is an option which is named default(cubrid memmon , with no option command) that shows top5 memory using modules and transactions. MMON_MODULE_DEFAULT_OPTION is for that. There is also an module option named brief(cubrid memmon -m <b or brief>) that shows total memory usage of each module. MMON_MODULE_BRIEF_OPTION is for that.

Reply:
They are included in spec.